### DISCUSSION TITLE
Force playlist to start on time (FIXED)

### DISCUSSION THREAD

**Discussion by wizard-devcenter** [AzuraCast/AzuraCast#5326] [General]
Hello,

I have several playlists scheduled to start at a certain time. The problem is that the previous playlist goes over the next one, as the last song is not over. Is there a way to force the playlist to start on the right time ?

L.E.: Consider this one closed,

**Comment by gmantzak** [comment]
Hi. :)
Would it be possible please to share the solution? I am facing the same problem with Azuracast v0.17.4

I have two playlists with the following configuration and the previous one keeps playing at the scheduled time of the new one.
Thank you very much

1. Source: Song-Based 
2. Playlist Type: General Rotation
3. Song Playback Order: Shuffled 
4. Playlist Weight: 3-Default 
5. Advanced: Interrupt other songs to play at scheduled time.
